runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/configurationImplementations.kt

```diff
@@ -34,8 +34,8 @@ class GradleSourceRootImpl : SourceRoot, Serializable {
 }
 
 open class GradleDokkaSourceSet constructor(
-    @Transient @get:Input val name: String,
-    @Transient @get:Internal internal val project: Project
+    @get:JsonIgnore @Transient @get:Input val name: String,
+    @get:JsonIgnore @Transient @get:Internal internal val project: Project
```

Comment:
`@Transient` was used to indicate that those properties shouldn't be serialized by GSON. If Jackson ignores it, we can remove them and only leave the `JsonIgnore` IMHO (and everywhere below)

Reply:
Transient annotation is still necessary for Serializable to work (see the new test commitet 🎊 )